frameworks/MLPlan/exec.py

```diff
@@ -102,8 +103,9 @@ def run(dataset, config):
         probabilities=probabilities,
         probabilities_labels=probabilities_labels,
         target_is_encoded=is_classification,
```

Comment:
It looks like the target is not (no longer?) encoded. Running `python runbenchmark.py MLPlan openml/t/59 -f 0` fails due to `predictions` being a list with string labels (e.g. `Iris-Setosa`) while `target_is_encoded` is set to `True`.